_includes/headtags.html

```diff
@@ -1,14 +1,14 @@
 <meta charset="utf-8">
 <!-- Canonical and Alternate Links -->
-{% assign sorted_languages = site.languages | sort %}
-{% for lng in sorted_languages %}
-  {% if lng == site.default_lang %}
-    <link rel="canonical" href="{{ site.url }}{{ site.baseurl }}{{ page.url }}">
-  {% else %}
+{% comment %}
+English URLs don't have a lang subdirectory (exact reason unkown):
```

Comment:
```suggestion
English URLs don't have a lang subdirectory since it's the default language:
```

https://github.com/jamulussoftware/jamuluswebsite/blob/628943e60ad9c73d615ac913a101b40744052ce2/_config.yml#L6

Reply:
That doesn't explain why `en` doesn't appear in the URL, though -- it just says what the default value is.

Reply:
But the default is just no language folder I guess? 


We use https://github.com/untra/polyglot

Reply:
Well, that probably needs to be in the explanatory comment, in case things change in the tooling.

Reply:
@gilgongo Could you please add the link?